### DISCUSSION TITLE
Whats the difference between the two Expedition 3's?

### DISCUSSION THREAD

**Discussion by ghost** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#34] [Q&A] [answered]
Hi, just asking if there is any difference between those two files?

**Comment by cwmonkey** [comment, marked as answer]
The Redux version has rewards from seasons 1, 4 and 5. The non-redux version has the original season 3 rewards. Play the non-redux version if you intend to also play expeditions 1, 4 and 5 individually.

**Reply by ghost** [reply]
Thank you for the fast answer!
